pycbc/events/significance.py

```diff
@@ -192,6 +196,41 @@ def get_n_louder(back_stat, fore_stat, dec_facs,
         **kwargs)
 
 
+def get_far(back_stat, fore_stat, dec_facs,
+            background_time,
+            method=_default_opt_dict['method'],
+            return_counts=False,
+            **kwargs):  # pylint:disable=unused-argument
+    """
+    Return the appropriate FAR given the significance caluclation method
+    """
+    # Get n_louder in background and foreground according to the chosen
+    # method
+    back_cnum, fnlouder = get_n_louder(
+        back_stat,
+        fore_stat,
+        dec_facs,
+        method=method,
+        **kwargs
+    )
+
+    # If we are counting the number of louder events in the background,
+    # we add one. This is part of the p-value calculation in Usman 2015.
+    # If we are doing trigger fit extrapolation, this is not needed
+    if method == 'n_louder':
+        back_cnum += 1
+        fnlouder += 1
+
+    # Turn n_louder into a FAR by dividing by background time
+    back_far = back_cnum / background_time
```

Comment:
Neater to write bg_far .. gitlab doesn't seem to let me make a multi-line suggestion (?)
```suggestion
    bg_far = bg_n_louder / background_time
```